account_payment_return_import_iso20022/wizard/camt_parser.py

```diff
@@ -123,6 +126,8 @@ def parse_payment_returns(self, ns, node):
                 payment_return['transactions'].extend(transactions)
                 subno = 0
                 for transaction in payment_return['transactions']:
+                    if not transaction.get('date'):
+                        transaction['date'] = return_date
```

Comment:
For statements, is there not an `Ntry`date that would be relevant? Since a statement can contain entries for several dates.